src/app/(auth)/signup/page.tsx

```diff
@@ -105,34 +106,29 @@ export default function SignUp() {
             <BigButton type="button" onClick={handleSignUp}>
               Sign Up
             </BigButton>
-            <H4Centered>
+            <PCentered>
               Have an account already?{' '}
-              <LinkColored $color={COLORS.greyDark} href="/login">
+              <LinkColored $color={COLORS.blueMid} href="/login">
                 Log In
               </LinkColored>
-            </H4Centered>
+            </PCentered>
           </SpacerDiv>
         </>
       )}
       {emailSentCount > 0 && (
-        <SpacerDiv>
-          <H2>An email verification link has been sent.</H2>
-          <H4 $color={COLORS.greyDark}>
+        <SpacerDiv $align="center">
+          <Icon type="email" />
+          <AuthSubHeading>Verification email sent!</AuthSubHeading>
+          <H4Centered $color={COLORS.greyDark} $pad>
             This link will direct you to the next step. If you didn’t receive an
             email, please click Resend Email.
-          </H4>
-          <HorizontalDiv>
-            <Button
-              $primaryColor={COLORS.blueMid}
-              $secondaryColor={COLORS.blueDark}
-              onClick={handleResendEmail}
-            >
-              <H4 $color="white">Resend Email</H4>
-            </Button>
-            {emailSentCount > 1 && (
-              <P $color={COLORS.greyMid}>Email has been resent!</P>
-            )}
-          </HorizontalDiv>
+          </H4Centered>
+          <BigButton type="button" onClick={handleResendEmail}>
+            Resend Email
+          </BigButton>
+          {emailSentCount > 1 && (
+            <P $color={COLORS.blueMid}>Email has been resent!</P>
```

Comment:
could we add the email here?
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/01ec1460-0c41-41d3-85e2-94c5afbcafc8)